src/js/entities-service/entities/directories.js

```diff
@@ -0,0 +1,21 @@
+import BaseCollection from 'js/base/collection';
+import BaseModel from 'js/base/model';
+
+const TYPE = 'directories';
+
+const Model = BaseModel.extend({
+  type: TYPE,
+  url() {
+    return `/api/directory/${ this.get('slug') }`;
```

Comment:
Oh ffs 💢 we should fix this.

Reply:
Well at the moment we do need to be able to request via the slug for form definitions, but it could certainly be a `filter[slug]=` or the like

Reply:
No, I mean that we should use `/api/directories/{slug}`... or maybe not, since this is a unique case that uses `slug` instead of `uuid`.

Reply:
I'm ok either way at this point.   it basically just gets set here and forgotten